Hunting Queries/SecurityEvent/HostsWithNewLogons.yaml

```diff
@@ -19,14 +22,28 @@ query: |
   let endtime = todatetime('{{EndTimeISO}}');
```

Comment:
FYI only: I don't think this is actually going to work.... unless someone edits it manually :-( Not sure how to handle.